lib/commands/actions.js

```diff
@@ -0,0 +1,28 @@
+import androidHelpers from '../android-helpers';
+import log from '../logger';
+
+const swipeStepsPerSec = 28;
+
+let commands = {}, helpers = {}, extensions = {};
+
+commands.swipe = async function (startX, startY, endX, endY, duration, touchCount, elId) {
+  if (startX === 'null') {
```

Comment:
Should this check for both `null` and `undefined`? If you use [utils](https://github.com/appium/appium-support/blob/master/lib/util.js) from `appium-support` there is a method `hasValue` that checks.

```
import { util } from 'appium-support';

...

startX = util.hasValue(startX) ? startX : 0.5;
startY = util.hasValue(startY) ? startY : 0.5;
```